docs/lookups.rst

```diff
@@ -86,6 +87,27 @@ You can specify an output lookup with the following syntax::
 
   ConfVariable: ${output someStack::SomeOutput}
 
+
+.. _`default lookup`:
+
+default Lookup
+--------------
+
+The ``default`` lookup type will check if a value exists for the variable
```

Comment:
This isn't 100% true, right?  It can also be used to give default variables to lookups.

```
${default ${output MyVpc::VpcId}::vpc-123456}
```

Reply:
I... hadn't, really thought of that. It's an interesting idea. I didn't write the lookup module, so let me dig into it to check if that would work. (Or @troyready could chime in, since he wrote it)